server/src/main/java/org/apache/druid/client/indexing/ClientCompactionRunnerInfo.java

```diff
@@ -135,9 +135,8 @@ public static CompactionConfigValidationResult validatePartitionsSpecForMSQ(Part
     if (!(partitionsSpec instanceof DimensionRangePartitionsSpec
           || partitionsSpec instanceof DynamicPartitionsSpec)) {
       return CompactionConfigValidationResult.failure(
-          "MSQ: Invalid partitioning type[%s]. Must be either 'dynamic' or 'range'",
+          "MSQ: Invalid partitioning type[%s]. Must be either dynamic or range",
```

Comment:
I think having the quotes was better. You can even use square brackets instead of quotes.

Reply:
Reverted to quotes. When the text was passed as part of exception, single-quotes weren't being rendered in the web-console as `&apos;` -- which is not the case when passed as part of `AutoCompactionSnapshot`.